library/src/main/kotlin/com/chuckerteam/chucker/internal/ui/MainActivity.kt

```diff
@@ -248,6 +281,93 @@ internal class MainActivity :
             negativeButtonText = getString(R.string.chucker_cancel),
         )
 
+    private fun getSaveDialogData(
+        @StringRes dialogMessage: Int,
+    ): DialogData =
+        DialogData(
+            title = getString(R.string.chucker_save),
+            message = getString(dialogMessage),
+            positiveButtonText = getString(R.string.chucker_save),
+            negativeButtonText = getString(R.string.chucker_cancel),
+        )
+
+    private fun showSaveDialog(exportType: ExportType) {
+        showDialog(
+            getSaveDialogData(
+                when (exportType) {
+                    TEXT -> R.string.chucker_save_text_http_confirmation
+                    HAR -> R.string.chucker_save_har_http_confirmation
+                },
+            ),
+            onPositiveClick = {
+                when (exportType) {
+                    TEXT -> saveTextToFile.launch(EXPORT_TXT_FILE_NAME)
+                    HAR -> saveHarToFile.launch(EXPORT_HAR_FILE_NAME)
+                }
+            },
+            onNegativeClick = null,
```

Comment:
this is unneceesary?

Reply:
There is no action on dismiss (old actions also set this listener as null)